gcn/notices/km3net/alert.schema.json

```diff
@@ -0,0 +1,85 @@
+    },
+    {
+      "$ref": "../core/AdditionalInfo.schema.json",
+      "description": "Nb. of pipeline triggering on this event, targeted FAR and details on selection."
+    }
+  ],
+  "properties": {
+    "$schema": true,
+    "pipeline": {
+      "enum": [
+        "exceptional_evt_arca",
+        "exceptional_evt_orca",
+        "multiplet_arca",
+        "multiplet_orca"
+      ],
+      "description": "Pipeline issuing this notice: either single, exceptional (high-energy), event or multiple coincident events, from KM3NeT/ARCA or KM3NeT/ORCA data."
+    },
+    "p_value": {
+      "type": "number",
+      "description": "Occurence probability of the alert reported."
+    },
+    "src_error_50": {
+      "type": "number",
+      "description": "50% CL uncertainty region described as a circle with a radius [deg]."
+    },
+    "event_details": {
+      "type": "array",
+      "items": { "$ref": "#/$defs/km3event" },
+      "description": "List and details of KM3NeT events involved in the alert."
+    },
+    "potential_counterpart": {},
+    "counterpart_info": {
+      "type": ["array", "string"],
+      "items": { "$ref": "#/$defs/knownsrc" },
+      "description": "No search / No match / (not exhaustive) Array of known astrophysical sources or transient present in the KM3NeT event RoI."
+    }
+  },
+  "$defs": {
+    "km3event": {
+      "type": "object",
+      "allOf": [
+        { "$ref": "../core/Event.schema.json" },
+        { "$ref": "../core/DateTime.schema.json" },
+        { "$ref": "../core/Localization.schema.json" }
+      ],
+      "properties": {
+        "prob_evt_topology": {
+          "enum": ["track", "starting track", "shower"],
+          "description": "Best reconstructed topology, possible to acknowledge if track starts inside the detector volume."
+        },
+        "cos_zenith": {
+          "type": "number",
+          "description": "Value of the cosine of the angle between the event direction and the vertical: 1 correspond to up-going event, -1 is down-going."
+        },
+        "proxy_energy": {
+          "type": "number",
+          "description": "Estimation of the energy of the lepton track/shower, resulting from the neutrino interaction [GeV]."
+        }
+      },
+      "description": "[Id, Date_Event, RA, Dec, Radius_Error, Topology, Zenith, reco_energy] of the KM3NeT events involved in the alert."
+    },
+    "knownsrc": {
+      "type": "object",
+      "allOf": [
+        { "$ref": "../core/FollowUp.schema.json" },
+        { "$ref": "../core/Localization.schema.json" },
+        { "$ref": "../core/DateTime.schema.json" },
+        { "$ref": "../core/Distance.schema.json" },
+        { "$ref": "../core/Redshift.schema.json" }
+      ],
+      "description": "(not exhaustive) list of [reference, position and additional information] for sources founded in the KM3NeT event RoI"
```

Comment:
This doesn't need to list the properties, just describe that this is properties of known sources found in the KM3NeT event region of interest".  Best to spell out RoI, unless it's used multiple times.